src/Parameters.php

```diff
@@ -36,9 +36,10 @@ protected function decode($value)
         $decoded = '';
         $parts = imap_mime_header_decode($value);
         foreach ($parts as $part) {
-            $charset = 'default' == $part->charset ? 'auto' : $part->charset;
+            // $charset = 'default' == $part->charset ? 'auto' : $part->charset;
             // imap_utf8 doesn't seem to work properly, so use Transcoder instead
-            $decoded .= Transcoder::create()->transcode($part->text, $charset);
+            // $decoded .= Transcoder::create()->transcode($part->text, $charset);
+            $decoded .= $part->text;
```

Comment:
Like Message\Part also headers are affected: when the raw value is used, no test fails and [testEncoding7Bit](https://github.com/ddeboer/imap/blob/master/tests/MessageTest.php#L41) passes. @ddeboer do you have any test case that justify this piece of code?

Reply:
No, so let’s disable it for now.